src/test/java/core/mate/academy/service/MachineServiceTest.java

```diff
@@ -25,28 +25,28 @@ public static void beforeClass() {
     public void getAll_bulldozerType() {
         List<Machine> actual = machineService.getAll(Bulldozer.class);
         assertEquals("The list should contain elements of type Bulldozer",
-            Bulldozer.class, actual.get(0).getClass());
+                Bulldozer.class, actual.get(0).getClass());
```

Comment:
try to undo changes to test classes
review your changes before you commit to avoid unintended changes done with autoformatter or project-wide replacements